### PR TITLE
Add tests for Project Euler Problem 5 + minor refactor

### PR DESCRIPTION
**Pull request by pomkarnath98**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [ ] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [ ] All new algorithms have a URL in their comments that points to Wikipedia or another similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by pomkarnath98** [comment]
@raklaptudirm @appgurueu I have formatted the code with Prettier. But the Codecov report says, `2 lines in your changes missing coverage`. What are we following now to avoid this?

**Comment by appgurueu** [comment]
As the codecov report shows, the trivial error check is not tested (see https://app.codecov.io/gh/TheAlgorithms/JavaScript/pull/1691/blob/Project-Euler/Problem005.js). I would probably just remove it.

**Comment by pomkarnath98** [comment]
@appgurueu @raklaptudirm I think this PR now can be reviewed and merged.

**Review by raklaptudirm — DISMISSED** [review]

File: Project-Euler/test/Problem005.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { findSmallestMultiple } from '../Problem005.js'
+
+describe.concurrent('Find smallest multiple', () => {
+  test('if max divisor is 10', () => {
```
Comment (appgurueu) [inline Project-Euler/test/Problem005.test.js:4]: Could use `it.each` here

Comment (pomkarnath98) [reply Project-Euler/test/Problem005.test.js]: @appgurueu updated. Please check

**Review by appgurueu — APPROVED** [review]

**Pull request merged**